{"data_type": "CVE", "data_format": "MITRE", "data_version": "4.0", "CVE_data_meta": {"ID": "CVE-2018-1112", "ASSIGNER": "cve@mitre.org"}, "affects": {"vendor": {"vendor_data": [{"vendor_name": "gluster", "product": {"product_data": [{"product_name": "glusterfs", "version": {"version_data": [{"version_value": "3.3.0", "version_affected": "="}, {"version_value": "3.5", "version_affected": "="}, {"version_value": "4.0.2", "version_affected": "="}]}}]}}]}}, "problemtype": {"problemtype_data": [{"description": [{"lang": "en", "value": "CWE-264"}]}]}, "references": {"reference_data": [{"url": "https://access.redhat.com/articles/3422521", "name": "https://access.redhat.com/articles/3422521", "refsource": "CONFIRM", "tags": ["Third Party Advisory"]}, {"url": "https://access.redhat.com/errata/RHSA-2018:1268", "name": "RHSA-2018:1268", "refsource": "REDHAT", "tags": ["Third Party Advisory"]}, {"url": "https://access.redhat.com/errata/RHSA-2018:1269", "name": "RHSA-2018:1269", "refsource": "REDHAT", "tags": ["Third Party Advisory"]}, {"url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-1112", "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-1112", "refsource": "CONFIRM", "tags": ["Issue Tracking", "Third Party Advisory"]}, {"url": "https://review.gluster.org/#/c/19899/1..2", "name": "https://review.gluster.org/#/c/19899/1..2", "refsource": "CONFIRM", "tags": ["Vendor Advisory"]}]}, "description": {"description_data": [{"lang": "en", "value": 'glusterfs server before versions 3.10.12, 4.0.2 is vulnerable when using "auth.allow" option which allows any unauthenticated gluster client to connect from any network to mount gluster storage volumes. NOTE: this vulnerability exists because of a CVE-2018-1088 regression.'}]}}
